BencodeNET/Torrents/Torrent.cs

```diff
@@ -227,8 +227,8 @@ public virtual BDictionary ToBDictionary()
 
             var trackerCount = Trackers.Flatten().Count();
 
-            if (trackerCount > 0)
-                torrent[TorrentFields.Announce] = new BList(Trackers.First().Select(x => new BString(x, Encoding)));
+            if (trackerCount == 1)
+                torrent[TorrentFields.Announce] = new BString(Trackers.Flatten().Single(), Encoding);
```

Comment:
This should be changed to `Trackers.Flatten().First()` after reverting the `trackerCount` check above to avoid an exception if there are more than one tracker.